code401Challenges/src/test/java/code401Challenges/sort/MergeSortTest.java

```diff
@@ -0,0 +1,44 @@
+package code401Challenges.sort;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MergeSortTest {
+
```

Comment:
Good tests! One additional one I recommend is arrays with one longer than the other. That would test your last two while loops.